Gridy/View/WhiteLayerView.swift

```diff
@@ -26,15 +28,16 @@ class WhiteLayerView: UIView {
         let width = self.bounds.width
         let height = self.bounds.height
         var drawRectangle = CGRect()
-        let gridFrameView = GridFrameView()
+        let gridFrameView = imageEditorViewController.gridFrameView! // nil!!!
```

Comment:
It is nil because `ImageEditorViewController` must be initialized via a Storyboard and not via a nib file (the line 12).

Anyway, you don't want to create a new `ImageEditorViewController` object, you want to use the one on screen. As a clue, you want line 12 to be:
```
var imageEditorViewController: ImageEditorViewController?
```

Going further, the line 12 should simply be:
```
var gridFrameView: UIView?
// or the following for direct usage with Storyboard
@IBOutlet var gridFrameView: UIView!
```

Going even further, what I think is the cleanest implementation of the line 12:
```
var gridFrame: CGRect = .zero
```